proto/shentu/oracle/v1alpha1/oracle.proto

```diff
@@ -103,3 +103,18 @@ message TaskIDs {
 message CoinsProto {
     repeated cosmos.base.v1beta1.Coin coins = 1 [ (gogoproto.moretags) = "yaml:\"amount\"", (gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins" ];
 }
+
+message TxTask {
+    option (gogoproto.equal) = false;
+    option (gogoproto.goproto_getters) = false;
+    option (cosmos_proto.implements_interface) = "TaskI";
+
+    bytes tx_hash = 1 [ (gogoproto.moretags) = "yaml:\"tx_hash\"" ];
+    string creator = 2 [ (gogoproto.moretags) = "yaml:\"creator\"" ];
+    repeated cosmos.base.v1beta1.Coin bounty = 3 [ (gogoproto.moretags) = "yaml:\"bounty\"", (gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins" ];
+    google.protobuf.Timestamp valid_time = 4 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false, (gogoproto.moretags) = "yaml:\"valid_time\""];
+    google.protobuf.Timestamp expiration = 5 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false, (gogoproto.moretags) = "yaml:\"expiration\""];
```

Comment:
I think they can be merged, since `expiration_time` should always come before `valid_time` to be meaningful (because we can always set `expiration_time = valid_time` and have no functional difference on the oracle protocol) and expiration time shouldn't really be before valid_time (because we should wait a long as we can?)

Reply:
Yeah, I agree with you that `expiration_time ` should be after `valid_time `. 
My piece of thought is that a task could be kept in store for a while even after `valid_time ` is up. Because there may are delayed responses coming for this task, I mean, after the `valid_time`. In that case, with the existence of the task, we could correctly respond to the operator. Otherwise, we have to wrongly create a new task and expect an "overdue score request".

Reply:
`expiration_time` here is for fast path, right?  
If oracle module received task score from operator,  but never receive the score tx from endpoint , we can remove this task at `expiration_time`

Reply:
Yes, that may be another usage case for `expiration_time`. @xiumingdou